### DISCUSSION TITLE
How to get such a nice powershell?

### DISCUSSION THREAD

**Discussion by Jimmy2004** [microsoft/winget-cli#582] [General] [answered]
I noticed that the powershell has a theme that looks like the newest Microsoft edge. It is so beautiful. How can I get that powershell?

**Comment by ShirleyNekoDev** [comment, marked as answer]
That's the Windows Terminal.
https://github.com/microsoft/terminal

You can find it in the Microsoft Store:
[⇒ Windows Terminal Stable](https://www.microsoft.com/en-us/p/windows-terminal/9n0dx20hk701)
[⇒ Windows Terminal Preview](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n8g5rfz9xk3)

**Reply by denelon** [reply]
You can also `winget install Microsoft.WindowsTerminal`

If you have the msstore source enabled you can specify which source you want it to come from.
